README.md

```diff
@@ -112,22 +112,19 @@ for i in "IMAGE_CONJUR" "IMAGE_POSTGRES" "IMAGE_NGINX"; do
 done
 ```
 
-The Conjur data key is generated by the
-deployer and does not need to be created
-beforehand.
-
 #### Create namespace in your Kubernetes cluster
 
 We recommend running Conjur in its own namespace.
 If you use a different namespace than the `default`, run the command below to create a new namespace:
 
 ```shell
 kubectl create namespace "$NAMESPACE"
+kubectl config set-context --current --namespace="$NAMESPACE"
 ```
 
 #### Install the application with Helm to your Kubernetes cluster
 
-This project uses the upstream [cyberark/conjur-oss Helm chart](https://github.com/cyberark/conjur-oss-helm-chart).
+This project uses the upstream [cyberark/conjur-oss Helm chart](https://github.com/cyberark/conjur-oss-helm-chart). (You do not need to clone or helm install this repo directly; this will be done indirectly via the helm install of conjur below.)
 
 Use `helm` to deploy the application to your Kubernetes cluster:
```

Comment:
I think we should specify that the instructions are for Helm 2 only (and not for Helm 3)

Reply:
Yes, good point!